src/components/Attachments/AttachmentView/index.js

```diff
@@ -95,7 +96,7 @@ function AttachmentView({
     // For this check we use both source and file.name since temporary file source is a blob
     // both PDFs and images will appear as images when pasted into the the text field
     const isImage = Str.isImage(source);
```

Comment:
@aimane-chnaif I wonder why this isn't `ìsImage = Str.isImage(source) || (file && Str.isImage(file.name))` then next we can just use `if (isImage && isValidImage)`, am I missing something on why this could be intentional?

Reply:
I was actually just taking over the original logic from the previous code: https://github.com/margelo/expensify-app-fork/blob/9fa8c89b095d88b4066f209c16d34e3ee261e991/src/components/AttachmentView.js#L105-L106

I think the difference here is that we first check if the `source` is an image, else if there is a `file` prop and if that's an image. Not sure if this is intentional though, so i think somebody else has to answer this question. :)